src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageOutputPath>../../bin/nuget</PackageOutputPath>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+  </PropertyGroup>
+  <ItemGroup>
+    <Folder Include="Localization\transXliff\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SqlServer.SqlManagementObjects" Version="150.18040.0-preview" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.6.0" />
+    <PackageReference Include="Microsoft.SqlTools.Hosting" Version="1.0.0" />
```

Comment:
why is this a package reference instead of a project reference?

Reply:
There's a version conflict between the newly added ManagedBatchParser and these projects when we used project reference, So we decided to refer from NuGet.

Reply:
Please, explain what the conflict is about...

Reply:
a binary reference isn't the way to go here.  it will need to be constantly updated for one.  I'm not sure the error, but it appears there is some type of inconsistent SMO references.  Please refer to the other projects that are successfully referencing these projects using ProjectReference in this solution for examples.